src/extensions/lens-extension.ts

```diff
@@ -45,11 +45,12 @@ export class LensExtension {
   }
 
   getPageUrl(baseUrl = "") {
-    return compile(this.routePrefix)({ name: this.name }) + baseUrl;
+    const validUrlName = this.name.replace("@", "").replace("/", "-");
```

Comment:
Is there a uniqueness requirement for the name? This would have to be part of that because `@fred` would clash with one already named `fred`

Reply:
It won't clash since when package name with `@` the format is `@org/package-name`, not just `@some-name`

Reply:
I think we should be OK because I don't think `@fred` is a valid package name on NPM (and hopefully not on other registries like GH). You'd have to have a package name too, like `@fred/fred`, which would become `fred-fred` here, and wouldn't conflict with a package published as just `fred` (without a scope).

Reply:
I can see it is unlikely but it's theoretically possible to have mirantis-foo and @mirantis/foo

Reply:
What's the difference now between `getPageUrl` and `getPageRoute`? To me they look identical 🤔

Reply:
@jakolehm you've asked already similar question just some of days ago (page-registry `path` vs `url`) 😂
`getPageRoute` contain route placeholders like `:name`, etc.

Reply:
@ixrock `getPageRoute` calls `getPageUrl` and just appends `baseRoute` to it. Isn't that exactly what happens if `getPageUrl(baseRoute)` is called??

Reply:
They are the same when no additional (suffix) route provided for the page.
`PageRegistry`:
```
item.routePath = item.extension.getPageRoute(item.routePath)
```

Reply:
This is `baseRoute` of page-item, not baseRoute of extension.

Reply:
to me it looks like `this.getPageUrl('foo')` returns the same as `this.getPageRoute('foo')`

Reply:
Oh, now I see what you mean.. Both functions are similar but for different purposes..
Will refactor..

Reply:
@jakolehm PTAL, also fixed proper sub-pages route-path.